src/Mado/QueryBundle/Interfaces/EntityClass.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Mado\QueryBundle\Interfaces;
+
+interface EntityClass
+{
+    public function getFQNC();
```

Comment:
a better name?

Reply:
Is an idiomatic name that stands for FullyQualifiedClassName, is this better thatn FQCN?

Reply:
Maybe